Helper/CustomerSerializer.php

```diff
@@ -5,20 +5,25 @@
 class CustomerSerializer extends \Magento\Framework\App\Helper\AbstractHelper
 {
     public function __construct(
-        \Magento\Newsletter\Model\Subscriber $subscriberModel
+        \Magento\Newsletter\Model\Subscriber           $subscriberModel,
+        \Magento\Customer\Api\GroupRepositoryInterface $groupRepository
     ) {
         $this->subscriberModel = $subscriberModel;
+        $this->groupRepository = $groupRepository;
     }
     
     public function serialize($customer) {
         $this->subscriberModel->unsetData();
+        $group       = $this->groupRepository->getById($customer->getGroupId());
+        $groupName[] = $group->getCode();
```

Comment:
Can the customer belong only to one group?

Reply:
In Magento2 customer can belong only to one group without system modification.

Reply:
What if a customer doesn't belong to any group?

Reply:
This is not possible. Customer group is required for every customer.